src/include/pyptr.h

```diff
@@ -15,23 +15,25 @@ class PyPtr {
  public:
   PyPtr(O* o) : _obj(o) {}
 
-  O* operator->() { return _obj; }
+  PyPtr(const PyPtr& ptr) : _obj(ptr._obj) {
+    Py_IncRef((PyObject*)_obj);
+  }
 
-  operator O*() { return _obj; }
+   O* operator->() { return _obj; }
```

Comment:
Please add an explanatory comment here (I understand why you did this but it should be documented).

Reply:
I removed that operator actually, I just replaced it with a static cast

Reply:
Sam got rid of that, so I think it's resolved.